### ISSUE TITLE
is possible to deploy this nice theme on classic and old openwrt-14.07 , barrier breaker?

### ISSUE THREAD

**Issue by tacuar** [apollo-ng/luci-theme-darkmatter#13]
Really hope this very nice dark style theme  could be compiled in openwrt-14.07.

at this moment i've tried and it is not working out, having an error notice of no rule to compile the darkmatter. don't know why this would happen.

is possible to have some improvements and modifications so this could be used in BB?

or could you kindly give some tips about what we people can do who still love to use the old openwrt

**Comment by chron0** [comment]
hmm, haven't been on BB for a while. can you get luci-theme-bootstrap & luci-theme-material to work? If so, then it's probably just because of the lede changes that needed to be done to build with current upstream. You can try to copy the css into one of the themes you know to build or hack around in .mk files to make it work with old openwrt structure. in the end it shouldn't be too hard, since it's basically just a little css/js - nothing to compile really.

**Comment by tacuar** [comment]
Thank you so much : ) even i'm still not quite sure, but i will try it first and see if i'm able to work this out .

Once again, i really like this this theme so much, it makes OPENWRT have an enchanted appearance.

**Comment by tacuar** [comment]
and forgot to reply, i can get bootstrap work on bb, at this moment i've not tried material yet. Bootstrap is working very smoothly and very fast.

you said i could try copying some css into the themes that can work fine in BB, i guess you mean copy the css file of your darkmatter theme into the exising theme in BB, am i correct? just over write to the exising files.

OKey thank you so much and it gives me a light bulb in my head : )

**Comment by tacuar** [comment]
No rule to make target 'package/feeds/luci/luci/host/compile', needed by 'package/feeds/darkmatter/luci-theme-darkmatter/compile'.  Stop.

it seems not quite easy for making it to be usable in BB ..................

**Comment by chron0** [comment]
I've also no idea how many UI/CSS changes there were from BB to CC - if some item ids/names changed, it may cause weird effects.

It's never easy when you do almost anything for the first couple of times.
But doing/trying = training -> routine -> easy.

However: https://github.com/LuttyYang/luci-theme-material/issues/23 indicates there may be further problems ahead, since darkmatter is built on top of material. And why the hell are you even still hanging on BB? :)

**Comment by tacuar** [comment]
hi chron0, i stil hang on bb is for a specific wireless router which is very old, but is from that old router i started being a fan of openwrt and lede and wifi routers, it brings me lots of fun.

i have on my hands new devices but i still keep that first router that i played with.

It's not a serious problem but it's a kind of emotion, when i found something new, such as this darkmatter theme, i would like to decorate on my old router which is still running on BB : )

**Comment by tacuar** [comment]
i've given up and have to stay on the original bootstrap. Bootstrap responds super fast which is very good.

But i still install darkmatter on lateste LEDE, when all settings are done under bootstrap theme, then i switch to your darkmatter theme, it really looks so nice regarding to the color, layout and fonts.

**Comment by tacuar** [comment]
doing/trying = training -> routine -> easy.

i like this above quote, indeed.

while consuming lots of time, the outcome is great and lots of knowledge acquired.

**Comment by chron0** [comment]
yeah, time is inevitably something that needs to be invested. But I find it's worthwhile in many cases :) If I may find the time, I'll try to get rid of jquery and the material base dependency. That may be something I've picked up from lede dev crowd anyways, they were bitching (and rightly so) about darkmatters and materials jquery dep and hence its size. We can get it much smaller imho, perhaps then it may be even compatible to BB - or you start and beat me to it :)

**Comment by chron0** [comment]
Yes, the color/grad scheme of the "Darkmatter" series is something I've been working on refining for some time. People always think it's just a little colors/fonts, but getting them to align right in just the right amount has been quite time consuming. I often have myself and see many people with demands for darker UIs, so this was my approach to it.
